### PR TITLE
Dev 4

### PR DESCRIPTION
**Pull request by palashkaria**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -315,6 +315,20 @@ class Kronos extends Component {
     let input = e.target.value
     let datetime = Moment(input, this.format(), true)
     if (datetime.isValid()) {
+
+      // this changes is to create proper moment object with proper timezone.
+      // So that, it does not mess up the date to isoString convertion.
+      // In dropdown this.state.datetime is being used to calculate the time range,
+      // so we are using this moment object to create a new moment object for the
+      // choosen time/ date. {line no. : 324~333}
+      if (this.props.format === 'DD/MM') {
```
Comment (palashkaria) [inline src/index.js]: @anindya-ghosh what about the year?

Comment (anindya-ghosh) [reply src/index.js]: Yes we should add that also. I fixed this only for the specific case.

Comment (anindya-ghosh) [reply src/index.js]: I think we should wait until that case arises. @palashkaria

Comment (palashkaria) [reply src/index.js]: @anindya-ghosh as discussed, rider's DoB could be one case

**Comment by Mayank1791989** [comment]
closing we have updated the build script, and now using master for build code.

**Pull request closed without merging**